src/io/eval.rs

```diff
@@ -238,13 +238,17 @@ where
                 encode.plaintext.clone()
             };
             log_mem(format!("plaintext at {}, {} computed", level, j));
+            // C_att, xL: new encoding of next level
```

Comment:
encodings is actually not limited to the attributes `att` but also contains the encoding of `t`